pkg/controller/migmigration/task.go

```diff
@@ -251,7 +261,7 @@ func (t *Task) Run() error {
 
 	// Run the current phase.
 	switch t.Phase {
-	case Created, Started:
+	case Created, Started, Rollback, RollbackComplete: //TODO: what actions needs to be done in Rollback-like steps?
```

Comment:
@djwhatle based on itineraries, I created these two steps (Rollback and RollbackComplete), currently I do see what to put into these steps. Could you give more context on what additional actions need to be executed on the Rollback scenario? Thanks